File: WC1/abcz2.c
#include <stdio.h>

int main() {
    int input;
    int marker = 1;
    
    scanf("%d", &input);
    
    if ( input % 2 == 0 ) {
        printf("alpha\n");
        marker = 0;
    }
    if ( input % 3 == 0 ) {
        printf("bravo\n");
        marker = 0;
    }
    if ( input % 5 == 0 ) {
        printf("charlie\n");
        marker = 0;
    }
    if ( marker == 1 ) {
        printf("zulu\n");
    }
    
    return 0;
}

File: WC4/PHP/abcz2.php
<?php
    $temp = true;
    $input = fgets(STDIN);
    
    if ( $input % 2 == 0 ) {
        echo "alpha" . PHP_EOL;
        $temp = false;
    }
    if ( $input % 3 == 0 ) {
        echo "bravo" . PHP_EOL;
        $temp = false;
    }
    if ( $input % 5 == 0 ) {
        echo "charlie" . PHP_EOL;
        $temp = false;
    }
    if ( $temp == true ) {
        echo "zulu" . PHP_EOL;
    }
?>
